_maps/configs/aegis.json

```diff
@@ -0,0 +1,39 @@
+{
+	"prefix": "SSV",
+	"map_name": "Aegis-class Long Term Care Ship",
+	"map_short_name": "Aegis-class",
+	"map_path": "_maps/shuttles/shiptest/syndicate_aegis.dmm",
+	"map_id": "syndicate_aegis",
+	"job_slots": {
+	"Captain": {
+			"outfit": "/datum/outfit/job/captain/syndicate",
+			"officer": true,
+			"slots": 1
+	},
+	"Lead Doctor": {
+		"outfit": "/datum/outfit/job/cmo/syndicate/nsv",
+		"slots": 1
+	},
+	"Ship Doctor":{
+		"outfit": "/datum/outfit/job/doctor/syndicate/nsv",
+		"slots": 2
+	},
+	"Mechanic": {
+		"outfit": "/datum/outfit/job/engineer/syndicate",
+		"slots": 1
+	},
+	"Deck Service Assistant": {
+		"outfit": "/datum/outfit/job/botanist/syndicate/nsv",
+		"slots": 3
```

Comment:
Suggestion: Reduce this to 2.
The kitchen/botany on this ship could be run by 2 people easily. And with cooking appliances only being circuits, it's probably better to increase to 3 during the game if needed, rather than drawing from other ship positions.